src/poker.js

```diff
@@ -1,5 +1,23 @@
 class Poker {
-  // Write your implementation here
+  winningPair(player1, player2) {
+    if (player1[0] !== player1[1] && player2[0] !== player2[1]) {
+      return [];
+    } else if (player1[0] === player1[1] && player2[0] !== player2[1]) {
+      return player1;
+    } else if (player1[0] !== player1[1] && player2[0] === player2[1]) {
+      return player2;
+    } else {
+      if (player1[0] > player2[0]) {
+        return player1;
+      } else {
+        return player2;
```

Comment:
nice concise code

Reply:
Or, you could write these if statements shorter.
`if (array1[0] !== array1[1] && array2[0] !== array2[1]) return []; if (array1[0] === array1[1] && array2[0] !== array2[1]) return array1; . . .`
and so on!